app/models/claim.rb

```diff
@@ -1,6 +1,8 @@
 class Claim < ApplicationRecord
   include ClaimDetails
 
+  before_destroy :destroy_attachments
+
```

Comment:
I think you made the right call here by adding this as a hook instead of, for example a concern added to the controllers where draft deletions happen. One thing we will need to remember/think about is if we were to go full app store, this functionality would need to be carried over accordingly - but that's almost a moot point as I can't see a reason to migrate drafts to app store in the future

Reply:
It'd be _far_ more effort than its worth, if it was ever a consideration I'd push back on it. We need to get local non-draft submissions migrated first before we can even consider it and that itself is a massive body of work.